ci/terraform/oidc/variables.tf

```diff
@@ -490,6 +489,12 @@ variable "orch_redirect_uri" {
   description = "The redirect URI set by Orchestration in the OAuth2 authorize request to Authentication"
 }
 
+variable "authorize_protected_subnet_enabled" {
```

Comment:
Do we want to try turning this on in sandpit?

Reply:
Sounds good, I'll turn that on :)